flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/connector/kafka/source/reader/deserializer/KafkaRecordDeserializationSchemaTest.java

```diff
@@ -53,7 +53,7 @@ public class KafkaRecordDeserializationSchemaTest {
     private static Map<String, ?> configuration;
     private static boolean isKeyDeserializer;
 
-    @Before
+    @BeforeEach
     public void setUp() {
```

Comment:
```suggestion
    void setUp() {
```
could be package private